packages/adblocker/src/filters/network.ts

```diff
@@ -888,6 +889,16 @@ export default class NetworkFilter implements IFilter {
             mask = setBit(mask, NETWORK_FILTER_MASK.isReplace);
             optionValue = value;
 
+            break;
+          case 'removeparam':
+            // TODO: Support regex
```

Comment:
How about `~` negation supported by AdGuard? I see it might be handled below but maybe worth documenting the limitation (and having a unit test for each)

Reply:
Leaving a note: from their documentation (Inversion section under `$removeparam` modifier; https://adguard.com/kb/general/ad-filtering/create-own-filters/#removeparam-modifier), it's a different one.

Reply:
It's now supported [`200515e` (#4528)](https://github.com/ghostery/adblocker/pull/4528/commits/200515e7d126b0f258eab13d47a445a86ff0323a)